tests/integration_tests/test_collection_embedding_function.py

```
@@ -384,6 +412,523 @@ def test_get_or_create_collection_both_none_error(self, db_client):
+
+    def test_providing_embedding_function_from_parameter_and_persistence_raises_error(
+        self, db_client
+    ):
+        """
+        Test that providing embedding function from parameter and persistence raises error.
+
+        Automatically runs for: embedded, server, oceanbase
+        """
+        collection_name = f"test_ef_parameter_and_persistence_{int(time.time() * 1000)}"
+        print(
+            f"\n✅ Testing providing embedding function from parameter and persistence raises error"
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)